libs/data-mapper-v2/src/components/functionConfigurationMenu/inputDropdown/InputDropdown.tsx

```diff
@@ -269,6 +272,12 @@ export const InputDropdown = ({
     }
   };
 
+  useEffect(() => {
+    if (inputValue === undefined) {
+      setCustomValue(undefined);
+    }
+  }, [inputValue]);
+
```

Comment:
Looks like there is another useEffect dependent on "inputValue". Can you merge all the useEffects where possible please? Having different useEffects dependent on the same will open us up for potential timing issues and future bugs.